content/middlewares.md

```diff
@@ -241,3 +241,5 @@ const app = await NestFactory.create(AppModule);
 app.use(logger);
 await app.listen(3000);
 ```
+
+> info **Hint** Accessing the DI container in a global middleware is not possible. You need to use a functional middleware instead when using `app.use()`. Alternatively, you can use a class middleware and consume it with `.forRoutes('*')`.
```

Comment:
```suggestion
> info **Hint** Accessing the DI container in a global middleware is not possible. You can use a functional middleware instead of using `app.use()`. Alternatively, you can use a class middleware and consume it with `.forRoutes('*')`.
```

Could we also link to the `functional middleware` section above?

Reply:
Yes, good idea linking it.

Reply:
I feel like the suggested change does not reflect what's right.

It's really that you _need to_ use a functional middleware with `app.use()`. It's not possible to use a class middleware there. Or how would this work?

Reply:
Ah, I see I initially misread what was being conveyed. Your change is fine, and the intent behind it is as intended. Ignore my suggestion :smile_cat: